--- gnome-control-center-3.24.0/configure~	2017-03-20 12:00:47.000000000 +0000
+++ gnome-control-center-3.24.0/configure	2017-04-09 17:23:18.920556373 +0000
@@ -18334,7 +18334,7 @@
 	have_networkmanager=yes
 fi
 if test "x$have_networkmanager" = xno ; then
-        linux_error_or_warn "*** Network panel will not be built (NetworkManager or ModemManager not found) ***"
+        echo "*** Network panel will not be built (NetworkManager or ModemManager not found) ***"
 else
 
 $as_echo "#define BUILD_NETWORK 1" >>confdefs.h
--- gnome-control-center-3.24.0/configure~	2017-04-09 17:23:18.000000000 +0000
+++ gnome-control-center-3.24.0/configure	2017-04-09 17:25:33.888768268 +0000
@@ -18649,7 +18649,7 @@
 $as_echo "#define BUILD_BLUETOOTH 1" >>confdefs.h
 
 else
-	linux_usb_error_or_warn "*** Bluetooth panel will not be built"
+	echo "*** Bluetooth panel will not be built"
 fi
  if test x$have_bluetooth = xyes; then
   BUILD_BLUETOOTH_TRUE=
--- gnome-control-center-3.24.0/configure~	2017-04-09 17:25:33.000000000 +0000
+++ gnome-control-center-3.24.0/configure	2017-04-09 17:30:14.769116511 +0000
@@ -19042,7 +19042,7 @@
 $as_echo "#define HAVE_WACOM 1" >>confdefs.h
 
 else
-	linux_usb_error_or_warn "*** Wacom panel will not be built"
+	echo  "*** Wacom panel will not be built"
 fi
  if test x${have_wacom} = xyes; then
   BUILD_WACOM_TRUE=
